src/main/java/com/forgetemplatemod/blocks/BlockBase.java

```diff
@@ -15,8 +15,8 @@
 	
 	public BlockBase(Material material, String name) {
 		super(material);
-		
-		setUnlocalizedName(name);
+
+		setTranslationKey(name);
```

Comment:
What does this do?

Reply:
This method was renamed by forge, it does the same thing as before.